examples/next/docs/react/basic-example/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "react-example-basic",
+  "version": "next",
+  "private": true,
+  "homepage": "https://handsontable.com/.",
```

I've seen this in few files:
```suggestion
  "homepage": "https://handsontable.com/",
```

---

Done https://github.com/handsontable/handsontable/commit/1b1a87adaf45d42a05d34b4bc8c029da4a137052